encoding/text/marshal.go

```diff
@@ -453,3 +544,65 @@ func (ew *errWriter) WriteByte(b byte) error {
 	}
 	return ew.err
 }
+
+// indentWriter is helper for writing indented text
+type indentWriter struct {
+	w errWriter
```

Comment:
Rather than compose the `indentWriter` from an `errWriter`, let's inline the `errWriter`'s functionality into this struct.

Reply:
Done.